mne/_fiff/meas_info.py

```diff
@@ -998,6 +998,7 @@ def _check_helium_info(helium_info):
     return helium_info
 
 
+# TODO: Add fNIRS convention to loc
```

Comment:
Digging through I found that `dipole` is also not documented.
```suggestion
# TODO: Add fNIRS and dipole convention to loc
```
If you feel like documenting it, `[3:6]` stores the dipole orientation.